Checkout/Checkout/Source/Tokenisation/Error/TokenisationError+TokenRequest.swift

```diff
@@ -21,6 +22,8 @@ extension TokenisationError {
         return 3001
       case .applePayTokenInvalid:
         return 1100
+      case .userCancelled:
+        return 0
```

Comment:
Why 0 similar to other PR and not 1000 series ?

Reply:
Discussed in other PR. As @ehab-al-cko similarly raised the point that userCancelled is not really an error, but we need to deliver it as outcome, I feel more confident in the current path (not an error, but delivered inside an error, so stands out from standard)